gnovm/pkg/gnolang/ownership.go

```diff
@@ -333,6 +333,10 @@ func (tv *TypedValue) GetFirstObject(store Store) Object {
 		// something in it; in that case, ignore the base.  That will
 		// likely require maybe a preparation step in persistence
 		// ( or unlikely, a second type of ref-counting).
+		if refValue, ok := cv.Base.(RefValue); ok {
+			return store.GetObject(refValue.ObjectID)
+		}
+
```

Comment:
there is a commented GetBase() method for PointerValue, let's uncomment/fix and use that, in line with other calls to GetBase() in this function.

Reply:
otherwise looks good.

Reply:
Thanks; I made the change.